package.json

```diff
@@ -25,17 +28,20 @@
     "index.js"
   ],
   "license": "MIT",
+  "dependencies": {
+    "ember-cli-babel": "^5.1.5"
+  },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.3",
-    "coveralls": "^2.11.13",
-    "ember-cli": "2.9.1",
+    "coveralls": "^2.11.15",
+    "ember-cli": "^2.10.0",
```

Comment:
Very interesting. It makes me a little uncomfortable allowing flexibility on the minor bit given that ember-cli has an involved upgrade process. Perhaps we should consider nailing this down to an explicit version as per the usual npm-based upgrade process?